lib/server.js

```diff
@@ -99,9 +125,12 @@ exports.serve = function serve(
   port,
   _connect = require('connect'),
   _process = process,
-  ui
+  ui,
+  ssl,
+  sslKey,
+  sslCert
 ) {
-  const server = new Server(watcher, host, port, _connect, ui);
+  const server = new Server(watcher, host, port, _connect, ui, ssl, sslKey, sslCert);
```

Comment:
This kind of looks error prone. We should consider changing this signature to accept an object instead of individual parameters like these.

Reply:
Sounds good, although I suspect we don't need to block this PR on it.